network-api/networkapi/templates/partials/footer.html

```diff
@@ -55,6 +57,26 @@ <h5 class="h5-heading">More about us</h5>
             </ul>
           </div>
         </div>
+        <div class="row mt-4 mb-4 mb-md-0">
+          <div class="col-12">
+            <form action="{% url 'set_language' %}" method="post">{% csrf_token %}
```

Comment:
implementation based on https://docs.djangoproject.com/en/2.2/topics/i18n/translation/#the-set-language-redirect-view

Reply:
source code for `def set_language`: https://docs.djangoproject.com/en/2.2/_modules/django/views/i18n/